util/testkit/testkit.go

```diff
@@ -251,6 +252,16 @@ func (tk *TestKit) ExecToErr(sql string, args ...interface{}) error {
 	return err
 }
 
+// MustGetErrCode executes a sql statement and assert it's error code of the error.
+func (tk *TestKit) MustGetErrCode(sql string, errCode int) {
+	_, err := tk.Exec(sql)
+	tk.c.Assert(err, check.NotNil)
+	originErr := errors.Cause(err)
```

Comment:
if the `err` is throwed by Parser, line syntax error, can it use `errors.Cause` to get the originErr?

Reply:
Normally, it can get the original error of any error except those that don't implement `Cause()`. In terms of the parser error, `errors.Cause` does not take effects. In other words, `err` == `originErr`.